README.md

```diff
@@ -872,6 +872,36 @@ module.exports = {
 };
 ```
 
+##### `hashStrategy`
+
+Type: `'use-local-name' | 'auto'`
+Default: `'use-local-name'`
+
+Should local name be used when computing the hash.
+
+- `'use-local-name'` Each identifier in a module gets its own hash digest.
+- `'auto'` Identifiers from the same module shares the same hash digest if it's possible. Use this value to optimize the output for better GZIP or Brotli compression.
```

Comment:
After review, I think `'auto'` is not good name, `'auto'` sounds like it should be default value, yes, maybe this should be the default (as I written above), but let's found better name, I think we can do it more understandable, I suggest: `resource-path` and `resource-path-and-local-name`, the developer can immediately understand what is used from the name, instead of reading the documentation

Reply:
There's a chance a developer would set a `resource-path`, but the hashing would work as `resource-path-and-local-name` because of the automagic circuit breaker. That may lead to confusion, especially when copypasting the config partially into StackOverflow or GH issues.

Maybe `minimal-subset` or `infer-from-template` or something like that? Something that implies the automatic nature of the setting

_I'm not a big fan of bikeshedding and just will use your suggestion if we don't come up with a decision after a short while_

Reply:
Let's use `minimal-subset` :+1:

Reply:
Renamed the option enum values in 381d470729f54b74657650bb7eea7370ad9f7ad9